cpp/ql/src/Best Practices/Likely Errors/CommaBeforeMisleadingIndentation.qhelp

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+If the expression to the right of a comma operator starts at an earlier column than the expression to the left, then
+this suspicious indentation possibly indicates a logic error, caused by a typo that may escape visual inspection.
+</p>
+<warning>
+This query has medium precision because CodeQL currently does not distinguish between tabs and spaces in whitespace.
+Alerts may therefore flag code that appears readable for one value of tab size but not another.
```

Comment:
This is very useful information for anyone looking up why they're getting an alert on this new query (and so it should definitely be in here). But I don't think we have any precedent for how to phrase this in a qhelp file, so I hope the Docs team will be able to advise on this.

Reply:
A search of the repository revealed that `<warning>` is mostly used in older code, but that we are still using it. Judging by: https://codeql.github.com/codeql-query-help/cpp/cpp-missing-check-scanf/, we don't currently have any styling set up for this warning, so we might want to open an issue to add styling for warnings.

Suggest maybe:

```suggestion
This query has medium precision because CodeQL currently does not distinguish between tabs and spaces in whitespace.
If a file contains mixed tabs and spaces, alerts may highlight code that is correctly indented for one value of tab size but not for other tab sizes.
```